tests/unit/sagemaker/test_deserializers.py

```diff
@@ -23,3 +23,13 @@ def test_bytes_deserializer():
     result = deserializer.deserialize(io.BytesIO(b"[1, 2, 3]"), "application/json")
 
     assert result == b"[1, 2, 3]"
+
+
+def test_stream_deserializer():
+    deserializer = StreamDeserializer()
+
+    stream, content_type = deserializer.deserialize(io.BytesIO(b"[1, 2, 3]"), "application/json")
+    result = stream.read()
```

Comment:
should the stream get closed afterward? (per the docstring in the class implementation)

Reply:
Fixed